src/Quantum/MP.jl

```diff
@@ -110,6 +111,17 @@ end
 const MPS = MatrixProduct{State}
 const MPO = MatrixProduct{Operator}
 
+tensors(ψ::TensorNetwork{MatrixProduct{P,Infinite}}, args...) where {P<:Plug} =
+    throw(MethodError("You need to specify the site for an infinite MatrixProduct$P"))
+
+tensors(ψ::TensorNetwork{MatrixProduct{P,Infinite}}, site::Int, args...) where {P<:Plug} =
+    tensors(plug(ψ), ψ, mod1(site, length(ψ.tensors)), args...)
+
```

Comment:
This should already be provided by the `tensors` method in `Quantum.jl`. Why reimplement this?
```suggestion
```

Reply:
Well, this is for the specific case of an `Infinite` chain of tensors. For example, if we have two tensors that repeat indefinitely, then `tensors(3)` should return `tensors(1)`. This is different from the `Open` and `Periodic` boundary conditions.